net-im/swift/swift-4.0.2.ebuild

```diff
@@ -0,0 +1,162 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils scons-utils toolchain-funcs
+
+DESCRIPTION="An elegant, secure, adaptable and intuitive XMPP Client"
+HOMEPAGE="https://www.swift.im/"
+SRC_URI="https://swift.im/downloads/releases/${P}/${P}.tar.gz"
+
+LICENSE="BSD BSD-1 CC-BY-3.0 GPL-3 OFL-1.1"
+SLOT="0/4"
+KEYWORDS="~amd64 ~x86"
+IUSE="expat examples experimental ft gconf hunspell lua icu idn qt5 test tools unbound zeroconf"
+REQUIRED_USE="ft? ( experimental )
+	gconf? ( qt5 )
+	hunspell? ( qt5 )"
+
+RDEPEND="dev-lang/lua:=
+	dev-libs/boost:=
+	dev-libs/openssl:0=
+	sys-libs/zlib:=
+	expat? ( dev-libs/expat )
+	!expat? ( dev-libs/libxml2:2 )
+	experimental? ( dev-db/sqlite:3 )
+	ft? (   net-libs/libnatpmp
+		net-libs/miniupnpc:= )
+	hunspell? ( app-text/hunspell:= )
+	icu? ( dev-libs/icu:= )
+	gconf? ( gnome-base/gconf:2 )
+	idn? ( net-dns/libidn:= )
+	qt5? ( dev-qt/qtcore:5
+		dev-qt/qtdbus:5
+		dev-qt/qtgui:5
+		dev-qt/qtmultimedia:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtsvg:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtwebkit:5
+		dev-qt/qtx11extras:5 )
+	unbound? ( net-dns/unbound
+		net-libs/ldns )
+	zeroconf? ( net-dns/avahi )"
+
+DEPEND="qt5? ( dev-qt/linguist-tools:5 )
+	>=dev-util/scons-3.0.1-r3
+	${RDEPEND}"
+
+DOCS=(
+	"DEVELOPMENT.md"
+	"README.md"
+	"Swiften/ChangeLog.md"
+)
+
+PATCHES=(
+	"${FILESDIR}"/${P}-make-generated-files-handle-unicode-characters.patch
+	"${FILESDIR}"/${P}-qt-5.11-compat.patch
+)
+
+src_prepare() {
+	default
+
+	# Hack for finding qt system libs
+	mkdir "${T}"/qt || die
+	ln -s "${EPREFIX%/}"/usr/$(get_libdir)/qt5/bin "${T}"/qt/bin || die
+	ln -s "${EPREFIX%/}"/usr/$(get_libdir)/qt5 "${T}"/qt/lib || die
+	ln -s "${EPREFIX%/}"/usr/include/qt5 "${T}"/qt/include || die
+
+	# Remove parts of Swift, which a user don't want to compile
+	! use lua && rm -fr Sluift
```

Comment:
All `rm` calls need `|| die`. Which also means you need to use full `if ...; then ...; fi` form.

Reply:
Thanks, fixed.